src/js/components/TaskViewComponent.jsx

```diff
@@ -103,36 +103,30 @@ var TaskViewComponent = React.createClass({
 
   getButtons: function () {
     var selectedTasksLength = Object.keys(this.state.selectedTasks).length;
-
-    var refreshButtonClassSet = classNames({
-      "btn btn-sm btn-info": true,
-      "hidden": selectedTasksLength !== 0
-    });
-
-    var killButtonClassSet = classNames({
-      "btn btn-sm btn-info": true,
-      "hidden": selectedTasksLength === 0
-    });
-
-    return (
-      <div className="btn-group">
+    if (selectedTasksLength === 0) {
+      return (
         <button
-            className={refreshButtonClassSet}
+            className="btn btn-sm btn-default"
             onClick={this.handleRefresh}>
           ↻ Refresh
         </button>
-        <button
-            className={killButtonClassSet}
+      );
+    } else {
+      return (
+        <div className="btn-group">
+          <button
+            className="btn btn-sm btn-info"
```

Comment:
How is are our style conventions in the moment? Did we decided to do it this way?
Seems a little be confusing to have the attributes of the button and the content at one column aligned.